bin/registerServerTopology.php

```diff
@@ -76,7 +76,18 @@
              - PROXY_PASSWORD           <optional> string
 
 EOD;
-
+try {
+    foreach ($opt as $key => $option) {
+        if (is_array($option)) {
+            throw new \InvalidArgumentException("
+                each flag must be declared only once : -$key as been declared twice or more
```

Comment:
```suggestion
                each flag must be declared only once : $key has been declared more than once
```
no dash should be displayed has you don't distinguish single and double dashes : -h vs --root